lib/logstash/config/mixin.rb

```diff
@@ -444,6 +445,12 @@ def validate_value(value, validator)
             end
 
             result = value.first
+          when :bytes
+            begin
+              result = Filesize.from(value.first).to_i
+            rescue ArgumentError
+              return false, "Unparseable filesize: #{value.first}"
+            end
```

Comment:
Could be a good idea to support no units at all and treat the number as bytes? for backward compatibility?